templates/hazelcast5-existing-cluster/inventory_plan.yaml

```diff
@@ -0,0 +1,13 @@
+provisioner: existing-cluster
+
+cluster_name: <cluster_name>
+
+nodes:
+  - private_ip: <private_ip>
+    public_ip: <public_ip>
+    user: <ssh_username>
```

Comment:
do we only support key based ssh login to existing hosts? is this in the documentation?